config/default/manager_auth_proxy_patch.yaml

```diff
@@ -22,6 +22,3 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
-      - name: operator
-        args:
-        - "--metrics-bind-address=127.0.0.1:8080"
```

Comment:
Why delete this?

Reply:
I thought it was added to add the patch to find the context of where auth proxy container has to be added. But it works without it. Also `metrics-bind-address` is set to `127.0.0.1:8080` by default.